liquidprompt

```diff
@@ -1550,6 +1557,77 @@ _lp_http_proxy_color() {
     lp_http_proxy_color="${LP_COLOR_PROXY}${LP_MARK_PROXY}${NO_COL}"
 }
 
+_lp_env_vars() { # [color_set, [color_unset]]
```

Comment:
This function needs docs.

I think I've been forgetting to check that recently, you might want to do a pass on recently merged PRs...

Reply:
I checked:
 - error meaning — OK
 - LP_MARK_JOBS_SEPARATOR — OK
 - CMAKE — OK
 - _lp_join — OK